homeassistant/components/mqtt/strings.json

```diff
@@ -605,7 +605,7 @@
                 "direction_value_template": "Direction value template"
               },
               "data_description": {
-                "direction_command_topic": "The MQTT topic to publish commands to change the fan direction payload, either `forward` or `reverse`. Use the direction command template to customize the payload. [Learn more.]({url}#direction_command_topic)",
```

Comment:
I think it is important to know for the user what payloads are published to the topic.

Reply:
Because there is no payload options for `forward` and `reverse`, I prefer to keep this in the comment, even when this means that information is repeated.